ci/unit/test_main.sh

```diff
@@ -34,7 +34,7 @@ rm -f "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/control"
 mkdir -p "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/"
 ln -s ../../../../../../../../src/control \
   "${SL_BUILD_DIR}/src/control/src/github.com/daos-stack/daos/src/control"
-DAOS_BASE=${SL_PREFIX%/install*}
+DAOS_BASE=${SL_SRC_DIR}
```

Comment:
There are potentially a couple of other cases which could switch from SL_PREFIX to SL_SRC_DIR as well but I'm not sure on the shell syntax they're using.